splatter.py

```diff
@@ -0,0 +1,32 @@
+
+import uuid
+from pymongo import MongoClient
+
+api_key_length = 30
+
+
+class SplatterService(object):
```

Comment:
Let's keep naming schemes for the top level objects/services/whatever to `Splat`thing so `SplatService`